docs/source/persistent_volumes.rst

```diff
@@ -0,0 +1,35 @@
+==================
+Persistent Volumes
+==================
+
+What are Persistent Volumes?
+----------------------------
+Persistent Volumes are a Kubernetes feature that allows you to attach stateful storage (like an EBS volume) to Pods (i.e., your PaaSTA instance).
+
+Caveats
+-------
+In general, we discourage the use of Persistent Volumes in favor of totally stateless services (i.e., where the state is separated from the service itself in a DB, S3, etc.)
+
+That said, there are several things to keep in mind before deciding to use Persistent Volumes:
+   - PaaSTA does not provide monitoring for Persistent Volumes - you are responsible for staying on top of your usage (i.e., there is no alerting for full or almost full volumes)
+   - Persistent Volumes cannot be resized online - if you run out of space and need to expand your volume: there *will* be downtime
+      - Additionally, PaaSTA cannot automatically handle this resize: you *will* need to find an engineer on the Compute Infrastructure team to do this
```

Comment:
Not sure if it's also worth noting that if they would like to delete their pv, they will have to contact Compute infra for that

Reply:
added :)